bin/check-beanstalk-elb-metric.rb

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env ruby
+#
+# check-cloudwatch-alarm
+#
+# DESCRIPTION:
+#   This plugin finds the desired ELB in a beanstalk environment and queries
+#   for the requested cloudwatch metric for that ELB
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#   ./check-beanstalk-elb-metric -e MyAppEnv -m Latency -c 100
+#
+# NOTES:
+#
+# LICENSE:
+#   Andrew Matheny
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugins-aws/cloudwatch-common'
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class BeanstalkELBCheck < Sensu::Plugin::Check::CLI
+  option :environment,
+         description: 'Application environment name',
+         short: '-e ENVIRONMENT_NAME',
+         long: '--environment ENVIRONMENT_NAME',
+         required: true
+
+  option :elb_idx,
+         description: 'Index of ELB.  Useful for multiple ELB environments',
+         short: '-i ELB_NUM',
+         long: '--elb-idx ELB_NUM',
+         default: 0,
+         proc: proc(&:to_i)
+
+  option :metric_name,
+         description: 'ELB CloudWatch Metric',
+         short: '-m METRIC_NAME',
+         long: '--metric METRIC_NAME',
+         required: true
+
+  option :period,
+         description: 'CloudWatch metric statistics period. Must be a multiple of 60',
+         short:       '-p N',
+         long:        '--period SECONDS',
+         default:     60,
+         proc:        proc(&:to_i)
+
+  option :statistics,
+         short:       '-s N',
+         long:        '--statistics NAME',
+         default:     'Average',
+         description: 'CloudWatch statistics method'
+
+  option :unit,
+         short:       '-u UNIT',
+         long:        '--unit UNIT',
+         description: 'CloudWatch metric unit'
+
```

Comment:
can you be consistent about the spacing here. there's really no need to align the values.